documentation/slate/source/includes/apis.md

```diff
@@ -1763,7 +1923,7 @@ For example:
 -   Is there any perturbations on the RER network ?
     -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/traffic_reports>
 -   Is there any perturbations on the "RER A" line ?
-    -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/lines/line:TRN:DUA810801043/traffic_reports>
+    -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/lines/line:OIF:810:AOIF741/line_reports?>
```

Comment:
Same here (that's a copy'n'paste...)

Reply:
Mistake: it should be  `traffic`_reports at the end